src/platforms/react-native/performance/instrumentation/user-interaction-instrumentation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: User Interaction Instrumentation
+sidebar_order: 20
+sdk: sentry.javascript.react-native
+
+description: "Learn what kinds of user interactions may be captured"
+---
+
+The UI instrumentation captures transactions and adds breadcrumbs for touch interactions. Gesture support using React Native Gesture Handler is also available with the `sentryTraceGesture` wrapper. 
+
+## Touch Events
+
+Transaction names are composed from a combination of displayed screen names, (for example, `LoginScreen`), and the labels of the element users interacted with, (for example, `login_button`).
+
+<Note>
+Capturing UI transactions requires that you first set up <Link to="/platforms/react-native/performance/instrumentation/automatic-instrumentation/#enable-routing-instrumentation">routing instrumentation</Link>.
+</Note>
+
+The UI instrumentation tracing is disabled by default, but you can enable it by setting the `enableUserInteractionTracing` options to `true` and wrapping your root component:
+
+```js
+import * as Sentry from '@sentry/react-native'
+
+Sentry.init({
+  // ...
+  integrations: [
+    new Sentry.ReactNativeTracing({
+      enableUserInteractionTracing: true,
+      routingInstrumentation,
+      // ...
+    }),
+  ],
+})
+
+const App = () => <View>Your App</View>
+
+export default Sentry.wrap(App)
+```
+
+The label by which UI elements are identified is set by the `labelName` option in `Sentry.wrap`. If no value is supplied, `sentry-label` will be used instead. If an element can't be identified, the transaction won't be captured.
+
+```js
+export default Sentry.wrap(App, { touchEventBoundaryProps: { labelName: 'my-label' } });
+```
+
+<Note>
+If the UI transaction idled, but didn't have any child spans, it'll be dropped.
+</Note>
+
+Because transactions are automatically bound to scope, you can create child spans using [manual instrumentation](/platforms/react-native/performance/instrumentation/custom-instrumentation/#retrieve-a-transaction). Note that the `idleTimeoout` for running UI transactions defaults to `1000` milliseconds (one second).
```

Comment:
```suggestion
Because transactions are automatically bound to scope, you can create child spans using [manual instrumentation](/platforms/react-native/performance/instrumentation/custom-instrumentation/#retrieve-a-transaction). Note, that the `idleTimeoout` for running UI transactions defaults to `1000` milliseconds (one second).
```